src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -130,6 +130,51 @@ static Object* arraySpeciesCreate(ExecutionState& state, Object* originalArray,
     return Object::construct(state, C, 1, argv);
 }
 
+// http://ecma-international.org/ecma-262/10.0/#sec-flattenintoarray
+// FlattenIntoArray(target, source, sourceLen, start, depth [ , mapperFunction, thisArg ])
+static int64_t flattenIntoArray(ExecutionState& state, Value target, size_t argc, Value* argv, bool isNewExpression)
+{
+    assert(target.isObject());
+    assert(argv[0].isObject());
+    assert(argv[1].toInteger(state) >= 0);
+    assert(argv[2].toInteger(state) >= 0);
+    assert(argv[3].isNumber());
```

Comment:
We defined `ASSERT` macro to enable `assert` statement only for debug mode. So, please fix these asserts by using the `ASSERT` macro.